index.html

```diff
@@ -93,14 +93,21 @@
                     </div>
                 </div>
                 <div class="team-card" id="filip">
-                    <img src="./assets/person.png" alt="Filip" class="team-image">
+                    <img src="./assets/filip_photo.jpg" alt="Filip" class="team-image" width="230px">
```

Comment:
bolo to potrebne? width:100% height: auto na class team-image by malo byt postacujuce